scripts/build_demand_profiles.py

```diff
@@ -176,8 +191,42 @@ def build_demand_profiles(
     substation_lv_i = n.buses.index[n.buses["substation_lv"]]
     regions = gpd.read_file(regions).set_index("name").reindex(substation_lv_i)
     load_paths = load_paths
-    # Merge load .nc files: https://stackoverflow.com/questions/47226429/join-merge-multiple-netcdf-files-using-xarray
-    gegis_load = xr.open_mfdataset(load_paths, combine="nested")
+
+    if str(load_paths[0]).endswith(".csv"):
```

Comment:
It would be nice to place this in an auxiliary function.
what do you think?

Regarding the read_csv, please, use the read_csv_nafix from helpers; it already supports additional arguments.
You could do:
`read_csv_nafix(pth, sep=";", dtype={...})`

moreover, I see that you create load_regions, but then you use only the region_name.
Do we need to define the dictionary?
I'm wondering if this work:
        gegis_load = gegis_load.assign_coords(
            {
                "region_name": ("region_code", df.region_name) [optionally with unique?]
            }
        )

Reply:
Agree, revised